pages/additionalresources.tsx

```diff
@@ -0,0 +1,49 @@
+import { Container } from "../components/bootstrap"
+import { createPage } from "../components/page"
+import AdditionalResourcesCard from "../components/AdditionalResources/AdditionalResourcesCard"
+import AdditionalResourcesCardContent from "../components/AdditionalResources/AdditionalResourcesCardContent"
+import styles from "../components/AdditionalResources/AdditionalResourcesCard.module.css"
+
+const content = [
+  {
+    paragraph: {
+      title: ``,
+      P1: `The MA Legislature has an online tool you can use to identify your legislators based on your home address.  https://malegislature.gov/Search/FindMyLegislator `
```

Comment:
Hey Jeff, so react automatically escapes strings for security reasons, which is why html in here doesn't end up rendered properly. Instead, we can specify the content as JSX directly. I would also remove the title since I don't think it's used here:

```
const content = [
<>The MA Legislature has an <b>online tool</b>...</>,
<>The MA Legislature publishes...</>,
<>Mass Legal Services...</>
]
```

Then we can pass the `value` directly to the content component:
```
            <AdditionalResourcesCardContent key={index}>
              {value}
            </AdditionalResourcesCardContent>
```

Reply:
@jkoren

Reply:
@alexjball great.  I put title back in because Typescript gave me an error message without it.  If you give me some guidance on removing the Typescript requirement, I can delete the titles.